railgun/config/crd/bases/configuration.konghq.com_kongplugins.yaml

```diff
@@ -0,0 +1,186 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.4.1
+  creationTimestamp: null
+  name: kongplugins.configuration.konghq.com
+spec:
+  group: configuration.konghq.com
+  names:
+    kind: KongPlugin
+    listKind: KongPluginList
+    plural: kongplugins
+    singular: kongplugin
+  scope: Namespaced
+  versions:
+  - name: v1
+    schema:
+      openAPIV3Schema:
+        description: KongPlugin is the Schema for the kongplugins API
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          config:
+            description: Config contains the plugin configuration.
```

Comment:
Do we use these generated CRDs yet? Generation now succeeds with the risky types option, but should still warn about this, and still generates a broken end result here (there's only the description, it needs `type: object` to function properly).

Not sure if we're just generating these but only using the generated UDPIngress CRD in practice because kubebuilder always generates everything.

Reply:
You're right, for the moment we're not using them, and it's fair I think to keep them out of this scope since this PR is meant to be the lightest touch UDP implementation just to get something out there for us to iterate further on.

As such I removed the non-UDP ones in 04f9fc148769b625dd84884c6990754456f52c73